script.js

```diff
@@ -192,6 +193,46 @@ document.getElementById('btn-hide').addEventListener('click', () => {
   document.getElementById('modal').style = 'display: none';
 });
 
-document.getElementById('btn-disappear').addEventListener('click', () => {
-  document.getElementById('modal').style = 'display: none';
-});
\ No newline at end of file
+// form validation
+
+const IsLowerCase = (Check) => /[A-Z]/g.test(Check);
+const Email = document.querySelector('#eInput');
+const form = document.querySelector('#form');
+const errorElement = document.querySelector('#ErrorMessages');
+const Name = document.querySelector('.fullName');
+// Email.style.borderColor = '#D0D9D4';
+// Name.style.borderColor = '#D0D9D4';
+
+form.addEventListener('submit', (e) => {
```

Comment:
Nice job with the features :heavy_check_mark: Every requirement was checked off. :clap: 

- [ ] Please, clean up your codebase of commented lines of code. It's important to keep a professional repository plus commented codes will make your codebase messy.